### ISSUE TITLE
lob stream error on expand on valueset on baseDstu3

### ISSUE THREAD

**Issue by jvitrifork** [jamesagnew/hapi-fhir#2034]
Following request: `http://hapi.fhir.org/baseDstu3/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.10.1.4--20171231000000/$expand` fails with the error message: 

```
{
  "resourceType": "OperationOutcome",
  "text": {
    "status": "generated",
  "issue": [ {
    "severity": "error",
    "code": "processing",
    "diagnostics": "Failed to call access method: org.springframework.orm.jpa.JpaSystemException: Unable to access lob stream; nested exception is org.hibernate.HibernateException: Unable to access lob stream"
  } ]
}
```

**Comment by jvitrifork** [comment]
The same goes for a random valueset on R4, eg. `http://hapi.fhir.org/baseR4/ValueSet/40608/$expand`

**Comment by jamesagnew** [comment]
Thanks for the heads up - This is already fixed on a working branch, but that branch isn't going to be merged for another week or two because of other issues.

I'll cherry pick this now so that we can get the fix into master for the 5.1.0 release.

**Comment by jamesagnew** [comment]
Fixed in f3fbad63e71dc225389766ffe07695c12ee30285
